Gruntfile.js

```diff
@@ -194,30 +205,24 @@ module.exports = function(grunt) {
 
 
         // Use Amazon S3 for images
-        s3: {
+        aws_s3: {
           options: {
-            key: '<%= secrets.s3.key %>', // define this in secrets.json
-            secret: '<%= secrets.s3.secret %>', // define this in secrets.json
-            access: 'public-read',
-            region: 'us-east-1', // feel free to change this
-            headers: {
-              // Two Year cache policy (1000 * 60 * 60 * 24 * 730)
-              "Cache-Control": "max-age=630720000, public",
-              "Expires": new Date(Date.now() + 63072000000).toUTCString()
-            }
+            accessKeyId: '<%= secrets.s3.key %>', // See README for secrets.json
+            secretAccessKey: '<%= secrets.s3.secret %>', // See README for secrets.json
+            region: '', // Enter region or leave blank for US Standard region
+            uploadConcurrency: 5, // 5 simultaneous uploads
+            downloadConcurrency: 5 // 5 simultaneous downloads
           },
-          dev: {
+          prod: {
             options: {
-              maxOperations: 20,
-              bucket: 'BUCKET-NAME' // define this
-            },
-            upload: [
-              {
-                src: 'dist/**/*',
-                dest: 'public/emails/', // define this
-                rel: 'dist', // rel must be set to maintain directory structure of src
-                options: { gzip: true }
+              bucket: '<%= secrets.s3.bucketname %>', // Define your S3 bucket name in secrets.json
+              differential: true, // Only uploads the files that have changed
+              params: {
+                CacheControl: '2000'
               }
+            },
+            files: [
+              {expand: true, cwd: '<%= paths.dist_img %>', src: ['**'], dest: 'test/<%= paths.dist_img %>'}
```

Comment:
No, it shouldn't be there. I actually made a modification just now to allow for subdirectories within the bucket (or it can be left blank to use the bucket at the main directory).